charts/cluster-api-provider-aws/templates/crd-awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
```

Comment:
Those are special annotations added for core OCP , we don't use that for the MCE deploy . I do recommend use the yaml under the config/default dir to avoid the special annotations that OCP team added. This is applied for all crds

Reply:
I have addressed it.